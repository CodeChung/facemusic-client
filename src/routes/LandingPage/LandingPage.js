import React from 'react'
import TokenService from '../../services/token-service';
import Home from '../../components/Home/Home';

class LandingPage extends React.Component {
    render() {
        const view = TokenService.hasAuthToken() ? <Home/> : <div>Yolo</div>
        return (
            <div>
                {view}
            </div>
        )
    }
}

export default LandingPage