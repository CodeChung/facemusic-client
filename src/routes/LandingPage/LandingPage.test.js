import React from 'react';
import ReactDOM from 'react-dom';
import LandingPage from './LandingPage';
import { BrowserRouter } from 'react-router-dom';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

describe(`LandingPage Component`, () => {
   it('renders without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(<BrowserRouter><LandingPage /></BrowserRouter>, div);
        ReactDOM.unmountComponentAtNode(div);
    }); 

    it('renders as expected', () => {
        const wrapper = shallow(<LandingPage />)
        expect(toJson(wrapper)).toMatchSnapshot()
    })
})