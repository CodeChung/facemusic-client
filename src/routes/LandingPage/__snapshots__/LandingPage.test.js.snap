// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandingPage Component renders as expected 1`] = `
<div
  className="LandingPage"
>
  <LandingInfo />
</div>
`;
